portafly/portafly_api.yml

```diff
@@ -56,47 +55,127 @@ paths:
         schema:
           type: integer
         in: query
+      - name: state
+        description: Account state.
+        schema:
+          type: string
+        in: query
+      responses:
+        "200":
+          $ref: '#/components/responses/developerAccountList'
+      operationId: getDeveloperAccounts
+      summary: Get developer accounts
+      description: Get a list with all developer accounts.
 components:
   schemas:
-    DeveloperAccount:
-      title: Root Type for developerAccountData
-      description: A simplified format for a developer account data.
+    Application:
+      title: Application
+      description: Represents an application.
       type: object
       properties:
         id:
-          description: The account ID number
+          description: The application ID number
           type: integer
-        created_at:
+        name:
+          description: The name of the application
+          type: string
+        state:
+          description: The current state of the application
+          type: string
+        account:
+          $ref: '#/components/schemas/DeveloperAccount'
+        plan:
+          $ref: '#/components/schemas/Plan'
+        created_on:
           format: date-time
           description: When it was created
           type: string
-        updated_at:
+        traffic_on:
           format: date-time
-          description: When it was last updated
+          description: ??
```

Comment:
What happened here, typo?

Reply:
It happened I didn't know what to put there.